src/paths.ts

```diff
@@ -6,7 +6,9 @@ import * as path from 'path';
 import {decode} from 'urlencode';
 
 /** converts a server-side XDebug file URI to a local path for VS Code with respect to source root settings */
-export function convertDebuggerPathToClient(fileUri: string|url.Url, localSourceRoot?: string, serverSourceRoot?: string): string {
+export function convertDebuggerPathToClient(fileUri: string|url.Url, pathMapping?: { [index: string]: string; }): string {
+    let localSourceRoot: string = '';
+    let serverSourceRoot: string = '';
```

Comment:
if these aren't initialized, then I receive an error testing for `serverSourceRoot && localSourceRoot` "variable is used before it is assigned"